src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -175,6 +178,9 @@ public async Task ProcessInputAsync(Message message, Flow flow, CancellationToke
                         // Create trace instances, if required
                         var (stateTrace, stateStopwatch) = _traceManager.CreateStateTrace(inputTrace, state);
 
+                        // Redirect to Attendance Action
+                        await _attendanceRedirectHandler.RedirectToAttendanceAsync(message.From.ToIdentity(), userIdentity, context, cancellationToken);
```

Comment:
The FlowManager class should not know anything about the specific flow actions ("Attendence" in the case). Move to another scope (outside?)